src/test/java/edu/oakland/test/database/DatabaseCommManagerTest.java

```diff
@@ -0,0 +1,92 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommManagerImplementation;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database CommManager Test")
+public class DatabaseCommManagerTest {
+
+  @Test
+  @DisplayName("rfid going in is Rfid coming Out")
+  void rfidInIsRfidOut() {
+
+    int i = 10;
+
+    DatabasePersistentStorageStub dpss = new DatabasePersistentStorageStub();
+    assertEquals(i, dpss.locateRfidData());
```

Comment:
Our goal here is to test the DatabaseCommManager, so we need to be calling that class in every one of our tests. An example of how to do that for this would be as follows:
```suggestion
    DatabasePersistentStorageStub dpss = new DatabasePersistentStorageStub(i);
    DatabaseCommManager dcm = new DatabaseCommManagerImplementation(dpss);
    assertEquals(i, dcm.passGetRfidRequest());
```